dev/NavigationView/NavigationView.cpp

```diff
@@ -1833,7 +1913,7 @@ void NavigationView::KeyboardFocusFirstItemFromItem(const winrt::NavigationViewI
 
     if (auto controlFirst = firstElement.try_as<winrt::Control>())
     {
-        controlFirst.Focus(winrt::FocusState::Keyboard);
+        SetFocus(controlFirst, winrt::FocusState::Keyboard);
```

Comment:
>Keyboard [](start = 50, length = 8)

should this be programmatic?

Reply:
see https://github.com/microsoft/microsoft-ui-xaml/pull/1850#discussion_r368166014